dts/bindings/rtc/nordic,nrf-rtc.yaml

```diff
@@ -22,4 +22,14 @@ properties:
       description: mmio register space
       generation: define
       category: required
+
```

Comment:
Missing binding for `prescaler` property.

Reply:
it's defined in rtc.yaml

Reply:
Right, I missed it. But its description is not adequate for Nordic peripherals. Perhaps we should not include `rtc.yaml` but instead define the bindings that we actually need?

Reply:
maybe i will just update description in `rtc.yaml` to something more generic. Like `Prescaler value determines RTC  frequency` and i can add comment like `Exact relation between prescaler and frequecy is vendor specific`

Reply:
not including this yaml will lead to redundancy of other fields.

Reply:
Actually, the common `prescaler` property could be used also for Nordic RTC nodes. You'd only need to use the value of `(32768 - DT_..._PRESCALER)` in the driver.

Reply:
Apparently, I misunderstood the purpose of the `prescaler` property in RTC nodes. It seems to be in fact the clock divider. The description does not really help...

Reply:
Ok, we discussed offline with @anangl and concluded with following:
- updated description in `rtc.yaml` which was wrong, changed to `RTC frequency equals clock-frequency divided by the prescaler value`.
- changed rtc prescaler in nordic devices to be 1 by default. Prescaler register gets `DT_PRESCALER - 1`

Reply:
> - updated description in rtc.yaml which was wrong, changed to RTC frequency equals clock-frequency divided by the prescaler value.

But the actual frequency will be CLK / (1 << PRESCALER_REG), so this description is not true...